modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -214,8 +213,8 @@ public Swift5ClientCodegen() {
         typeMapping.put("float", "Float");
         typeMapping.put("number", "Double");
         typeMapping.put("double", "Double");
-        typeMapping.put("file", "URL");
```

Comment:
Why did you change the `file` and `binary` formats from `URL` to `Data`?
This will break the existing generator.

Reply:
`file` and `binary` both refer to binary format and `URL` != binary.

Reply:
@wing328  @4brunu I noticed that there is inconsistency between `typeMapping` and `isDataTypeFile/isDataTypeBinary`. I believe that the typeMapping for both `file` and `binary` **should** be mapped to `Data` in Swift. [OpenAPI 3.0 file upload specs](https://swagger.io/docs/specification/describing-request-body/file-upload/)

Should I fix this and put it in a separate PR?

```
    typeMapping.put("file", "URL");
    typeMapping.put("binary", "URL");

    ....

    @Override
    public boolean isDataTypeFile(String dataType) {
        return "URL".equals(dataType);
    }

    @Override
    public boolean isDataTypeBinary(final String dataType) {
        return "Data".equals(dataType);
    }
```

Reply:
The `file` being associated with `URL` is right, because the `file` upload works based on the `file` `URL`.
The `binary` maybe it should be associated to `Data`, but I'm not 100% sure.

Reply:
From what I understand from the [2.0 specs](https://swagger.io/specification/v2/), `file` should *also* be associated with `Data`. You could try it yourself. Go to [Petstore](https://petstore.swagger.io/#/pet/uploadFile), open your browser's devtools and go to the network tab. Try the uploadFile endpoint and choose an arbitrary id and file then see what your request looks like.

The request has a `content-type` of `multipart/form-data` and the body contains the raw binary octets representing the file being uploaded. This is the same as the `binary` type except that `file` uses `multipart/form-data` and `binary` uses other MIME types like `application/octet-stream`.

Reply:
The Java and Kotlin generators also map `file` to `java.io.File`, and the `URL` is the closest we have to file in Swift.
Then the URLSession and Alamofire will configure the `multipart/form-data` with the file `URL`.